resources/lang/en/messages.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
 return [
-    
+    'successfully'      => 'City was create successfully!',
```

Comment:
The key name is `successfully` but the value is for only `City`, it's not good. please change